test/index.test.js

```diff
@@ -241,8 +241,8 @@ test('should handle Next initialization errors', async t => {
       prepare: () => Promise.reject(error)
     })
   })
-
-  t.rejects(() => Fastify().register(plugin), error)
+  const fastify = Fastify().register(plugin)
```

Comment:
why would this behavior change? I guess the test was checking that registering the plugin rejected, not awaiting the start of the server